### PR TITLE
feat(docker): add fallocate to built image

### PR DESCRIPTION
**Pull request by michalxo**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -32,6 +32,7 @@ LABEL name="Red Hat Messagign QE - Java CLI Image" \
 
 USER root
 
+RUN microdnf install util-linux -y
```
Comment (jiridanek) [inline Dockerfile:35]: Comment! Why do we install util-linux?

Probably does not matter too much, but a fancy way of running microdnf is (as shown e.g. https://github.com/skupperproject/skupper-router/blob/main/Containerfile#L22)
```suggestion
# install fallocate for use by tests
RUN microdnf -y --setopt=install_weak_deps=0 --setopt=tsflags=nodocs install util-linux && microdnf clean all -y
```

**Comment by jiridanek** [comment]
https://www.conventionalcommits.org/en/v1.0.0/#specification FTW, afaik @enkeys started using that recently... Certainly commit message should be in present tense imperative mood...

File: Dockerfile [hunk 2]
```diff
@@ -32,6 +32,9 @@ LABEL name="Red Hat Messagign QE - Java CLI Image" \
 
 USER root
 
+#Install fallocate util to create files of specific size
```
Comment (jiridanek) [inline Dockerfile:35]: I don't like this comment. People can be assumed to know what fallocate does. We need to say why we need it to have in the image.

> If I grep this entire project, nothing uses no fallocate. So, can I delete it?

And the answer should be an obvious no, thanks to this comment.

File: Dockerfile [hunk 3]
```diff
@@ -32,6 +32,9 @@ LABEL name="Red Hat Messagign QE - Java CLI Image" \
...
+RUN microdnf -y install \
+    util-linux
```
Comment (jiridanek) [inline Dockerfile:37]: I don't insist on `--setopt=install_weak_deps=0 --setopt=tsflags=nodocs`, but `&& microdnf clean all -y` probably should be there. Fedora dnf cache has like 80 MB

**Review by jiridanek — CHANGES_REQUESTED** [review]
pls fix comment and do the clean all thing

File: Dockerfile (hunk 3 above)
Comment (enkeys) [reply Dockerfile]: I do, please follow the best practise from UBI, `microdnf --setopt=install_weak_deps=0 --setopt=tsflags=nodocs install -y package && microdnf clean all `.

**Review by jiridanek — APPROVED** [review]
approved, squash and rebase (with github button) pls

**Review by enkeys — APPROVED** [review]

**Pull request merged**